src/ahrens_lab_to_nwb/yu_mu_cell_2019/yu_mu_cell_2019_conversion_script_dual_color.py

```diff
@@ -55,17 +68,35 @@
     # ActivityStates=dict(folder_path=str(states_folder_path), sampling_frequency=behavior_rate),
 )
 conversion_options = dict(
-    Imaging=dict(stub_test=True, stub_frames=3),
-    GliaSegmentation=dict(include_roi_centroids=False, mask_type="voxel"),
-    NeuronSegmentation=dict(include_roi_centroids=False, mask_type="voxel"),
+    Imaging=dict(stub_test=stub_test, stub_frames=stub_frames),
+    GliaSegmentation=dict(include_roi_centroids=False, mask_type="voxel", stub_test=stub_test, stub_frames=stub_frames),
+    NeuronSegmentation=dict(
+        include_roi_centroids=False, mask_type="voxel", stub_test=stub_test, stub_frames=stub_frames
+    ),
 )
 
 converter = YuMuCell2019NWBConverter(source_data=source_data)
 
+# Add synchronized timestamps to all imaging and segmentation objects
+with h5py.File(name=processed_behavior_file_path) as file:
+    frame_tracker = file["data"]["frame"][:]
+timestamps = np.where(np.diff(frame_tracker))[1][:-1] / behavior_rate
```

Comment:
I have not time to dwell the reasoning behind it but numpy recommends using nonzero instead of where for this cases:
https://numpy.org/doc/stable/reference/generated/numpy.where.html

Reply:
OK - yeah IDK what they mean about subclasses there, but if `where` is just a wrapper around it anyway I can use it directly